Firmware/planner.cpp

```diff
@@ -1008,17 +1008,17 @@ Having the real displacement of the head, we can calculate the total movement le
   // block->step_event_count ... event count of the fastest axis
   // block->millimeters ... Euclidian length of the XYZ movement or the E length, if no XYZ movement.
   float steps_per_mm = block->step_event_count.wide/block->millimeters;
+  uint32_t accel;
   if(block->steps_x.wide == 0 && block->steps_y.wide == 0 && block->steps_z.wide == 0)
   {
-    block->acceleration_st = ceil(cs.retract_acceleration * steps_per_mm); // convert to: acceleration steps/sec^2
+    accel = ceil(cs.retract_acceleration * steps_per_mm); // convert to: acceleration steps/sec^2
```

Comment:
Is the `ceil()` call actually correct? Ceil is to find the closest higher integral number, i.e. `ceil(1.1) == 2`.